build/Makefile.in

```diff
@@ -454,9 +454,6 @@ install: all
 	$(CP) src/strings/*.h $(DESTDIR)$(PREFIX)/include/moar/strings
 	$(CP) src/jit/*.h $(DESTDIR)$(PREFIX)/include/moar/jit
 	$(CP) src/instrument/*.h $(DESTDIR)$(PREFIX)/include/moar/instrument
-	$(MKPATH) $(DESTDIR)$(PREFIX)/include/libuv
-	$(MKPATH) $(DESTDIR)$(PREFIX)/include/libtommath
-	$(CP) 3rdparty/libuv/include/*.h $(DESTDIR)$(PREFIX)/include/libuv
```

Comment:
Actually, having these line in Makefile.in creates the directories unconditionally. Also it's a duplication of what gets created via Configure.pl already - sans the ```$(MKPATH) ... libtommath``` thingy, which i just added to the Configure.pl.
(I am talking of the non-pkgconfig case here. In case of pkgconfig these are not needed at all.)